### PR TITLE
travis: use the 3.7 debian packages

### PR DESCRIPTION
**Pull request by juhaszviktor**
Signed-off-by: Juhász Viktor <viktor.juhasz@balabit.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,20 @@ language: c
...
   - sudo apt-get install -qq curl
-  - wget http://download.opensuse.org/repositories/home:/tusavik/xUbuntu_12.04/Release.key
+  - wget http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key
   - sudo apt-key add Release.key
-  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/tusavik/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
+  - rm -rf Release.key
+  - wget http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/Release.key
+  - sudo apt-key add Release.key
+  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04 ./" >> /etc/apt/sources.list'
+  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
```
Comment (lbudai) [inline .travis.yml]: Are you sure in that you will need both repositories?

Comment (juhaszviktor) [inline .travis.yml]: Yes, becuase your repositories does not have the librdkafka-dev and
libgrok-dev packages

Comment (juhaszviktor) [inline .travis.yml]: And what is the problem with additional repositories?

Comment (lbudai) [inline .travis.yml]: I see, thx.

Comment (juhaszviktor) [inline .travis.yml]: Welcome! :)

**Pull request merged**